### PR TITLE
Added Helm Info at the top

### PR DESCRIPTION
**Pull request by othylmann**
Franz has been asked about helm, this could be a good place for that as a general 2 lines at the top which I added, also pointing people to the app catalog.

### CODE REVIEW

File: src/content/reference/app-configuration/index.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ type: subsection
 # App Configuration Reference
 
 Giant Swarm's [App Platform](/basics/app-platform/) allows you to easily install Apps across your entire
-fleet of clusters.
+fleet of clusters. We fully support Helm as a general tool to deploy your applications as well as for our general app catalogue, which you can of course also use for your own applications by creating a new catalogue.
```
Comment (marians) [inline src/content/reference/app-configuration/index.md:11]:
```suggestion
fleet of clusters. We fully support Helm as a general tool to deploy your applications as well as for our general app catalog, which you can of course also use for your own applications by creating a new catalog.
```

We use that spelling

**Comment by othylmann** [comment]
ok, this is clearly nothing for me. I have no clue what you mean.

**Comment by othylmann** [comment]
oh you changed catalog. I don't get the number thing

**Comment by jgsqware** [comment]
@othylmann https://github.com/giantswarm/docs/pull/469/files#diff-3a2b87683c3a269b890af50cd314897bL4

This line is metadata for for the document date.  Marian asked you to put the right date (today IIRC as it;s the updated date).

We cannot make a suggestion as it;'s not something you touch in the PR

**Comment by othylmann** [comment]
I think I added that date :) good now?

File: src/content/reference/app-configuration/index.md [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
-date: 2020-04-08
+date: 2020-04-27
 type: subsection
 ---
 
 # App Configuration Reference
 
 Giant Swarm's [App Platform](/basics/app-platform/) allows you to easily install Apps across your entire
-fleet of clusters.
+fleet of clusters. We fully support Helm as a general tool to deploy your applications as well as for our general app catalog, which you can of course also use for your own applications by creating a new catalog.
```
Comment (jgsqware) [inline src/content/reference/app-configuration/index.md:11]:
```suggestion
fleet of clusters. We fully support Helm as a general tool to deploy your applications as well as for our general app catalogue, which you can of course also use for your own applications by creating a new catalogue.
```

Comment (jgsqware) [reply src/content/reference/app-configuration/index.md:11]: Marian did a suggestion you didnt validate and it was outaded by your latest commit. 

So click on `Commit suggestion` to accept it up here.

**Comment by othylmann** [comment]
I think that's that :)

**Pull request merged**